config/settings.yml

```diff
@@ -12,6 +12,13 @@ financial:
     csv: true
     xml: true
 
+converters:
+  default:
+    product_to_journal_rows: Converters::ProductToJournalRows
+    order_detail_to_journal_rows: Converters::OrderDetailToJournalRows
```

Comment:
Part of me wants a more specific `JournalRowConverters` namespace (or something). `Converters` just seems really general for this use case.